root_orchestrator/docker-compose.yml

```diff
@@ -20,8 +20,10 @@ services:
       logging_jobname: "containerlogs"
     expose:
       - "10000"
+      - "50052"
     ports:
       - "10000:10000"
+      - "50052:50052"
```

Comment:
Isn't this port necessary in 1-DOC as well?

Reply:
Exactly: it's the `SYSTEM_MANAGER_GRPC_PORT`. Let me fix it 👍